include/deal.II/lac/dynamic_sparsity_pattern.h

```diff
@@ -446,6 +446,14 @@ class DynamicSparsityPattern : public Subscriptor
   compute_mmult_pattern(const SparsityPatternTypeLeft & left,
                         const SparsityPatternTypeRight &right);
 
+  /**
+   * Construct and store in this object the sparsity pattern corresponding to
+   * the product of transposed @p left and @p right sparsity pattern.
```

Comment:
To ensure unambiguity, should we write this as `transposed @p left and non-transpose @p right`, or something to that effect?